sdk/test-utils/recorder/README.md

```diff
@@ -109,53 +95,23 @@ Having the recorder as a devDependency means that you'll be able to start record
 The test-recorder provides the `Recorder` class that deals with recording and playing back the network requests, depending on the value assigned to the `TEST_MODE` environment variable.
 
 - If `TEST_MODE` equals to `record`, it will automatically store network requests in a plain text file in the folder `recordings` at the root of your library (it is `sdk/tables/data-tables` in our example).
-- This package assumes that the tests in the sdk are leveraging
-  [mocha](https://mochajs.org/) and [rollup](https://rollupjs.org/guide/en/)
-  (and [karma](https://karma-runner.github.io/latest/index.html) test runner for browser tests) as suggested by the [template](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/template/template) package in the repo.
+- As of version 4.0.0, this package supports `vitest` (instead of `mocha` and `karma`) for testing frameworks as suggested by the [template](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/template/template) package in the repo. And we employ `process.env` through vitest to access environment variables in both Node and browser environments.
```

Comment:
```suggestion
- As of version 4.0.0, this package supports `vitest` (instead of `mocha` and `karma`) for testing frameworks as suggested by the [template](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/template/template) package in the repo.
```
Looks it's repeated in later sections about `env`